athlinks-schema.ttl

```diff
@@ -195,81 +175,47 @@ athlinks:agePercentage
   rdfs:domain athlinks:IntervalResult;
   rdfs:range xsd:decimal .
 
-athlinks:isHidden
-  a owl:DataTypeProperty ;
+athlinks:elevation a owl:DataTypeProperty;
+  rdfs:label "Elevation";
+  rdfs:comment "Elevation";
+  rdfs:domain schema:Landform;
+  rdfs:range schema:QuantitativeValue.
+
+athlinks:altitude a owl:DataTypeProperty;
+  rdfs:label "Altitude";
+  rdfs:comment "Altitude";
+  rdfs:domain schema:Landform;
+  rdfs:range schema:QuantitativeValue.
+
+athlinks:elevationGrade a owl:DataTypeProperty;
+  rdfs:label "Elevation Grade";
+  rdfs:comment "Elevation Grade";
+  rdfs:domain schema:Landform;
+  rdfs:range schema:QuantitiveValue.
+
+athlinks:distanceUnit a owl:DataTypeProperty;
+  rdfs:label "Default Distance Unit";
+  rdfs:comment "Default Distance Unit";
   rdfs:domain athlinks:AthleticRace;
-  rdfs:range xsd:boolean .
+  rdfs:range schema:Text.
 
-athlinks:isVisible
-  a owl:DataTypeProperty ;
+athlinks:distanceDisplayUnit a owl:DataTypeProperty;
+  rdfs:label "Distance Unit to Display";
+  rdfs:comment "Distance Unit to Display";
   rdfs:domain athlinks:AthleticRace;
-  rdfs:range xsd:boolean .
+  rdfs:range schema:Text.
 
-athlinks:isEnabled
-  a owl:DataTypeProperty ;
+athlinks:createdBy a owl:DataTypeProperty;
+  rdfs:label "ID of User Who Created Race";
+  rdfs:comment "ID of User Who Created Race";
   rdfs:domain athlinks:AthleticRace;
-  rdfs:range xsd:boolean .
+  rdfs:range schema:creator.
```

Comment:
We don't need this property. schema:creator does the same thing.
Additionally. rdfs:range schema:creator is incorrect as schema:creator is a property.